docs/docs/guides/platform/weave-self-managed.md

```diff
@@ -367,13 +470,13 @@ When using more than one replica (W&B recommend a least 3 replicas), ensure to h
 extraEnv:
   WF_CLICKHOUSE_REPLICATED: "true"
 ```
-This has the same effect of `replicated: true` which in preview.
+This has the same effect as `replicated: true`.
 :::
 
 
-3. Set the `clusterName` in `values.yaml` to `weave_cluster`. If it is not, the database migration will fail.  
+3. Set the `clusterName` in `values.yaml` to `weave_cluster`. The database migration fails if you do not set this value correctly. 
```

Comment:
In this case, the cluster name will be defined by this [line](https://github.com/wandb/weave/pull/5350/files#diff-488e133752a995c0a14c732d0b4514b73c05ef2a5a2191ad80452e3b2f683201R317)